documentation/contributing/releases/index.markdown

```diff
@@ -0,0 +1,135 @@
+---
+author: tylerjw
+layout: page
+slug: releases
+title: Releases
+redirect_from: "/documentation/contributing/releases.html"
```

Comment:
i don't think we need this meta data... there isn't currently a page called releases.html that needs redirection

Reply:
removed.